Source/BlackHoles/BoostedBHInitialData.impl.hpp

```diff
@@ -59,46 +57,44 @@ BoostedBHInitialData::Aij(Coordinates<data_t> a_coords) const
 
 /* PRIVATE */
 
-template <class data_t>
-AMREX_GPU_DEVICE data_t
-BoostedBHInitialData::center_dist(Coordinates<data_t> a_coords) const
+[[nodiscard]] AMREX_FORCE_INLINE AMREX_GPU_DEVICE amrex::Real
+BoostedBHInitialData::center_dist(Coordinates a_coords) const
 {
-    data_t r = std::sqrt(std::pow(a_coords.x - m_params.center[0], 2) +
-                         std::pow(a_coords.y - m_params.center[1], 2) +
-                         std::pow(a_coords.z - m_params.center[2], 2));
+    amrex::Real r = std::sqrt(std::pow(a_coords.x - m_params.center[0], 2) +
+                              std::pow(a_coords.y - m_params.center[1], 2) +
+                              std::pow(a_coords.z - m_params.center[2], 2));
 
     double minimum_r = 1e-6;
-    return simd_max(r, minimum_r);
+    return (r < minimum_r) ? minimum_r : r;
```

Comment:
I missed this one. Sorry 🙁 
```suggestion
    std::max(r, minimum_r);
```